gnome-extra/evolution-data-server/evolution-data-server-3.46.0.ebuild

```diff
@@ -113,10 +113,10 @@ src_configure() {
 		-DENABLE_WEATHER=$(usex weather)
 		-DENABLE_LARGEFILE=ON
 		-DENABLE_VALA_BINDINGS=$(usex vala)
-		# Explicitly turn gtk4 off because gtk4webkit is not available yet
-		-DENABLE_GTK4=off
+		-DENABLE_GTK4=$(usex gtk)
+		# TODO this would pull in webkit2gtk:5.0, which is not available yet
+		-DENABLE_OAUTH2_WEBKITGTK4=OFF
```

Comment:
We have `webkit-gtk:4.1` and `webkit-gtk:5` now.

Reply:
probably better to do this with a use flag, to give users the chance to avoid pulling in another webkitgtk slot right?